sos/report/plugins/openstack_octavia.py

```diff
@@ -42,6 +42,7 @@ def setup(self):
             "/var/lib/octavia",
             self.var_config_data + "/octavia/etc/octavia",
             self.var_puppet_gen + "/etc/octavia",
+            self.var_puppet_gen + "/etc/octavia/rsyslog.d",
```

Comment:
Isn't this directory already collected by the previous line? sos report does collect whole path until sizelimit - or was the limit hit, in your case?

Reply:
Sorry the path was wrong. I updated the patch to point the correct path.